e2e/tests/test_e2e.py

```diff
@@ -0,0 +1,323 @@
+        k8s.wait_for_operator_pod_start()
+
+        jobs = k8s.get_logical_backup_job().items
+        actual_image = jobs[0].spec.job_template.spec.template.spec.containers[0].image
+        self.assertTrue(actual_image == image,
+                        "Expected job image {}, found {}".format(image, actual_image))
+
+        # delete the logical backup cron job
+        pg_patch_disable_backup = {
+            "spec": {
+               "enableLogicalBackup": False,
+            }
+        }
+        k8s.api.custom_objects_api.patch_namespaced_custom_object(
+            "acid.zalan.do", "v1", "default", "postgresqls", "acid-minimal-cluster", pg_patch_disable_backup)
+        k8s.wait_for_logical_backup_job_deletion()
+        jobs = k8s.get_logical_backup_job().items
+        self.assertTrue(0 == len(jobs),
+                        "Expected 0 logical backup jobs, found {}".format(len(jobs)))
+
+    def assert_master_is_unique(self, namespace='default', version="acid-minimal-cluster"):
+        """
+           Check that there is a single pod in the k8s cluster with the label "spilo-role=master"
+           To be called manually after operations that affect pods
+        """
+
+        k8s = self.k8s
+        labels = 'spilo-role=master,version=' + version
+
+        num_of_master_pods = k8s.count_pods_with_label(labels, namespace)
+        self.assertEqual(num_of_master_pods, 1, "Expected 1 master pod, found {}".format(num_of_master_pods))
+
+
+class K8sApi:
+
+    def __init__(self):
+
+        # https://github.com/kubernetes-client/python/issues/309
+        warnings.simplefilter("ignore", ResourceWarning)
+
+        self.config = config.load_kube_config()
+        self.k8s_client = client.ApiClient()
+
+        self.core_v1 = client.CoreV1Api()
+        self.apps_v1 = client.AppsV1Api()
+        self.batch_v1_beta1 = client.BatchV1beta1Api()
+        self.custom_objects_api = client.CustomObjectsApi()
+
+
+class K8s:
+    '''
+    Wraps around K8 api client and helper methods.
+    '''
+
+    def __init__(self):
+        self.api = K8sApi()
+
+    def get_spilo_nodes(self, pod_labels, namespace='default'):
+        master_pod_node = ''
+        replica_pod_nodes = []
+        podsList = self.api.core_v1.list_namespaced_pod(namespace, label_selector=pod_labels)
+        for pod in podsList.items:
+            if ('spilo-role', 'master') in pod.metadata.labels.items():
+                master_pod_node = pod.spec.node_name
+            elif ('spilo-role', 'replica') in pod.metadata.labels.items():
+                replica_pod_nodes.append(pod.spec.node_name)
+
+        return master_pod_node, replica_pod_nodes
+
+    def wait_for_operator_pod_start(self):
+        self. wait_for_pod_start("name=postgres-operator")
+        # HACK operator must register CRD / add existing PG clusters after pod start up
+        # for local execution ~ 10 seconds suffices
+        time.sleep(30)
```

Comment:
this is  sadly  not a  solution: [Travis build](https://travis-ci.org/zalando/postgres-operator/builds/536803211?utm_source=github_status&utm_medium=notification) fails but local one succeed.
does anyone has other ideas on how to circumvent the issue ?